Notes.md

```diff
@@ -1,4 +1,23 @@
 ##  Questions
+1- Can we do this as an exercise?
+So adding a non-optional attribute to ToDo, having to perform a migration.
+Do migrations. Priority from 0 (low) 1(med) 2(high) to its own entity with priority and color associated.
+
+2- I want to pass the filters from the view to the viewModel. Initially my idea was:
+struct FiltersView: View {
+ @EnvironmentObject var filters: TodoFilter
+  @State var viewModel: FiltersViewModel
+    init() {
+       viewModel = _viewModel = State(wrappedValue: FiltersViewModel(filters: filters)) // I can't get them here since self has not been initialized
+    }
+}
+}
+I endedup passing it into the viewModel on onAppear (FilterView / FilterViewModel).
+I know if I use coredata I could grab those filters easily using CoreDataStorage inside the viewModel, but the way I did, would you approach it differently?
```

Comment:
I tend to avoid environment objects and instead pass everything around explicitly. It makes the code less clean, but it's easier to do things like this.